README.md

```diff
@@ -32,30 +33,30 @@ docker run \
     --rm \
     --user="${UID}" \
     --volume="${PWD}:/github/workspace" \
-    ziti-cmake-test ./cmake.sh
+    ziti-builder-test ./ziti-builder.sh
 ```
 
 ### Publish the image to Docker Hub
 
-1. Create a pull request targeting the main branch.
-1. Merge the pull request to main.
+1. Create a pull request targeting the `main` branch.
+1. Merge the pull request to `main`.
 1. Create a meaningful release tag in GitHub that matches the regex `v[0-9]+.[0-9]+.[0-9]+`.
 1. The release will trigger a GitHub Action that builds and publishes the image to Docker Hub.
 
 ## Using this container image in a Ziti project
 
-Each Ziti project that uses CMake has a build script named `cmake.sh`. Each
-project's CI runs `cmake.sh` to build the default compilation target.
+Each Ziti project that uses this container image to build the project has a build script named `ziti-builder.sh`. Each
```

Comment:
this is most like not true. I would just list the tools that are available on the image and let consumer do whatever